src/core/dirlistjob.cpp

```diff
@@ -62,6 +63,12 @@ void DirListJob::exec() {
             false
     };
     if(enu) {
+        bool bindCutFiles = false;
+        if(cutFilesHashSet_
+                && cutFilesHashSet_->size() < 450 /* give up if more than 450 cut files */) {
```

Comment:
@vc-01 Could you please remove the condition about 450 files here and at FileInfoJob::exec()? It isn't needed anymore (there's no problem with thousands of files); moreover, it's confusing to the user.